app/controllers/api/admin/events_controller.rb

```diff
@@ -0,0 +1,2 @@
+class Api::Admin::EventsController < ApplicationController
+end
```

Comment:
Be sure to set your editor to add an extra newline at the end of every ruby file.  This is standard practice to help clean up diffs going forward. (if you add a new line of a text without an empty newline at the end of the file, the diff for your new change will be 2 lines instead of 1.)